lib/cli.js

```diff
@@ -2,9 +2,21 @@ const doc = `
 Usage: legofy
 
 Options:
-  -h, --help             Bring help.
-  -v, --version          Show version.
-  -f, --format=<format>  Set output format ('png' or 'jpeg') [default: jpeg].
+  -h, --help                   Bring help.
+  -v, --version                Show version.
+  -f, --format=<format>        Set output format ('png' or 'jpeg')
+                               [default: jpeg].
+  -m, --mode=<mode>            Preset for the performance/quality options
+                               [default: good].
+  --pixel-interval=<interval>  Set pixel interval for average color
+                               computation (higher means faster but less
+                               accurate).
+  --pattern-quality=<quality>  Canvas pattern quality.
+  --filter=<quality>           Canvas filter quality (same values as pattern
```

Comment:
This is a problem with `canvas` options naming then. But we can fix it on our side, maybe.